### PR TITLE
fix: Clicking listing closed modal on drop success modal 

### LINKED ISSUE #9670: Clicking listing closed modal on drop success modal
around 1m: 

https://github.com/kodadot/nft-gallery/assets/36627808/31da8de3-beb0-409b-9451-0dcac00dd192

### PR DESCRIPTION
**Pull request by hassnian**
### Context 

#9673 follow up

@daiagi  pls review

@prury pls test 

https://github.com/kodadot/nft-gallery/assets/44554284/3c50246a-9a7e-4628-816d-cdba3b1b3bea

### CODE REVIEW

File: composables/autoTeleport/useAutoTeleportTransitionDetails.ts [hunk 1]
```diff
@@ -91,9 +90,11 @@ export default function (
     () => !hasEnoughInCurrentChain.value,
   )
 
+  const forceActionAutoFees = computed(() => fees.forceActionAutoFees)
```
Comment (daiagi) [inline composables/autoTeleport/useAutoTeleportTransitionDetails.ts:93]: why extra `computed`?

can use `fees.forceActionAutoFees` directly right?

**Review by daiagi — CHANGES_REQUESTED** [review]
tested on monotifs drops

all worked ok (listing modal showed -> loaded -> listing completed successfully)

File: composables/autoTeleport/useAutoTeleportTransitionDetails.ts (hunk 1 above)
Comment (hassnian) [reply composables/autoTeleport/useAutoTeleportTransitionDetails.ts]: removed

**Review by daiagi — APPROVED** [review]

**Review by preschian — APPROVED** [review]
code lgtm

**Comment by yangwao** [comment]
@hassnian le'ts fix conflicts

**Pull request merged**